### ISSUE TITLE
Additional Party Details

### ISSUE THREAD

**Issue by tomhealey-icma** [finos/common-domain-model#2117]
I would like feedback on a proposal to add additional party details to the CDM. Currently, the party country is held in contactInformation or businessUnit->contactInformation but this is not the same as the party registered country. There are also many other party fields needed so I propose three options:
1) Since GLEI is widely accepted, I propose adding a new type to party, glei GleiRecord (*..1) that reflects the GLEI scheme. 
2) Add partyTaxonomy similar to productTaxonomy which would hold the GLEI records. 
3) Both of the above.

**Comment by chrisisla** [comment]
I don't think we should add anything specific to GLEI here. That is another data format which is always changing. So if we added a GLEI Record type to the model we would have to constantly update it to match whatever the new GLEI format is - plus there would be versioning issues between the CDM and the GLEI version.

Looking at the latest GLEI (https://www.gleif.org/en/about-lei/common-data-file-format/current-versions/level-1-data-lei-cdf-3-1-format) is there much detail in there that is missing from the model?

I am not adverse to adding a partyTaxonomy as it adds a lot more flexibility to what can be modelled for a party. However, we do run the risk of these taxonomies being abused, so we need to consider carefully whether to add one.

**Comment by tomhealey-icma** [comment]
The model currently has contract information and we need legal information such as address, jurisdiction, form and registration.
I'll go with adding party taxonomy similar to product taxonomy. If we can create an XML type then we can add schema format and reader as well.

**Comment by tomhealey-icma** [comment]
PR to be added for party Taxonomy.

**Comment by eteridvalishvili** [comment]
Hi @tomhealey-icma I hope you are doing well. Following up to see if the PR was added and if this issue can be closed.
